components/contributors/InfoCard.tsx

```diff
@@ -6,13 +6,13 @@ import Link from "next/link";
 import { BsSlack } from "react-icons/bs";
 import { env } from "@/env.mjs";
 
-export default function InfoCard({
+export default async function InfoCard({
   contributor,
-  minimal = true,
+  rank = 0,
```

Comment:
set it to `null` by default In situations where `rank` is null, we can hide it.

```suggestion
  rank,
```